fuel_test/env_network.yaml.default

```diff
@@ -0,0 +1,19 @@
+networking_parameters:
+  fixed_networks_vlan_start: 103
+  floating_ranges:
+  - - 172.16.0.128
```

Comment:
These don't make any sense as default settings - they are only useful for a specific host.  This either needs to be templated somehow or deleted with a script to generate the networks file based on parameters or an input file.